fiwaremurano/meta/io.murano/Resources/linux-init.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+service murano-agent stop
+
+AgentConfigBase64='%AGENT_CONFIG_BASE64%'
+
+if [ ! -d /etc/murano ]; then
+    mkdir /etc/murano
```

Comment:
One question here is, whats do you do if you do not have permissions to create a folder in /etc?